kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -173,6 +175,10 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
       val usersSet = users.split(",").toSet
       operations = operations.filter(operation => usersSet.contains(operation.getSession.user))
     }
+    if (StringUtils.isNotBlank(sessionHandle)) {
```

Comment:
Agree with @pan3793 , we can refactor this method to improve performance.

Reply:
Sure, i'll make a follow up PR fix those.